backend/api/chat.py

```
@@ -73,24 +71,30 @@ async def get_models_endpoint(_: None = Depends(verify_api_key)):
 @router.post("", response_model=ChatResponseModel)
 async def chat(
     request: ChatRequest,
-    _db_connection: DuckDBConnection = Depends(get_db_connection),
     chat_db: duckdb.DuckDBPyConnection = Depends(get_chat_db),
     config: BackendConfig = Depends(get_config),
     _: None = Depends(verify_api_key),
 ):
-    """LLMエージェント向けチャットエンドポイント。
+    """LLMエージェント向けチャットエンドポイント（薄いハンドラー）。
```

Ruff (0.14.11)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)